pkgs/tools/security/diceware/default.nix

```diff
@@ -7,28 +7,14 @@ with pythonPackages;
 
 buildPythonApplication rec {
   pname = "diceware";
-  version = "0.9.3";
+  version = "0.9.4";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0p09q6945qvdmvckjl8rfqx0g8nf6igc3c6rab6v74k9bsmbf15p";
+    sha256 = "0855n4dh16ws1dhw1ixxwk3wx7qr45xff7pn32pjg58p4z4cx168";
   };
 
-  nativeBuildInputs = [ pytestrunner ];
-
-  checkInputs = [ pytest ];
-
-  # NOTE: remove once 0.9.4 is released
-  patches = [
-    (fetchpatch {
-      url = "${meta.homepage}/commit/86379bf49ade2b486071d6d330515f01ecb06ab4.patch";
-      sha256 = "0nxvxiqvxfsa9y6zwy9k7shsd0fk92psdzi4klqwd4wy3lbmw8di";
-    })
-    (fetchpatch {
-      url = "${meta.homepage}/commit/a7d844df76cd4b95a717f21ef5aa6167477b6733.patch";
-      sha256 = "0ab4fc2pbl2hcxqw5rr6awbhlnmdna6igqjijywwr1byzb7ga4iq";
-    })
-  ];
+  nativeBuildInputs = [ coverage pytest pytestrunner ];
```

Comment:
Those should be `checkInputs` instead.

Reply:
I don't we think we could use these dependencies for cross compiling.

Reply:
However, `pytestrunner` needs to be in `nativeBuildInputs` or `buildInputs` since it is needed at build time, not at check time.

Reply:
Since `pytestrunner` would execute code of the target platform, `buildInputs` seems more appropriate. But I think nobody has yet looked into cross-compiling python packages.
I assume that for this package, `pytestrunner` is just wrongly put into `setup_requires`: https://github.com/ulif/diceware/blob/master/setup.py#L10

Reply:
That seems to be the standard way: https://github.com/pytest-dev/pytest-runner#example

Reply:
Ah, maybe the module is loaded in `setup.py` to provide a test subcommand or something.

Reply:
Yeah, it provides `setup.py pytest`.
@asymmetric I'd say just put `pytestrunner` it in `nativeBuildInputs` as it is done that way for other packages as well. We can change it later for all packages using `pytestrunner` at once.

Reply:
And move the rest to `checkInputs`?

Reply:
Yes